### ISSUE TITLE
When will Null safety be released

### ISSUE THREAD

**Issue by jeromekai** [Realank/flutter_datetime_picker#233]

**Comment by RamonJun** [comment]
+1

**Comment by nirbar89** [comment]
+2

**Comment by git2702** [comment]
+3

**Comment by ryojiro** [comment]
+4

**Comment by nguyendangkhoi** [comment]
+5

**Comment by git2702** [comment]
Do you have an ETA @Realank

**Comment by martinkong0806** [comment]
+6

**Comment by ClaudeChey** [comment]
+7

**Comment by Realank** [comment]
published in 1.5.1, sorry for delay

**Comment by yogithesymbian** [comment]
> published in 1.5.1, sorry for delay

still on error `flutter_datetime_picker: ^1.5.1`
```
: Warning: Operand of null-aware operation '??' has type 'Color' which excludes null.
../…/lib/flutter_datetime_picker.dart:311
 - 'Color' is from 'dart:ui'.
                  color: theme.backgroundColor ?? Colors.white,

```
